### ISSUE TITLE
Vor rose mode shows course as 000 when no course selected

### ISSUE THREAD

**Issue by BravoMike99** [flybywiresim/aircraft#8194]
### Aircraft Version

Development

### Build info

```json
{
    "built": "2023-09-04T02:30:14+00:00",
    "ref": "refs/heads/master",
    "sha": "12c8e357bccc95d58b6fab4d2e2051ca6b9940fb",
    "actor": "Benjozork",
    "event_name": "manual",
    "pretty_release_name": "master:12c8e357",
    "version": "v0.11.0-dev.12c8e35"
}
```

### Describe the bug

VOR course appears as 000 when no course selected 
![CRS](https://github.com/flybywiresim/aircraft/assets/119708186/25c9b41a-09ee-42a4-9b02-9eea352f5e65)

### Expected behavior

VOR course to be dashed

### Steps to reproduce

Have a VOR tuned and look at course in VOR ROSE

### References (optional)

https://youtu.be/fTDyNH6IL-M

### Discord Username (optional)

bruno_pt99

**Comment by tracernz** [comment]
That is correct. This will be corrected when I replace the MSFS NAV radio with our own systems. The MSFS nav radios do not have a possibility of no course.
